src/app/base/components/ZoneSelect/ZoneSelect.test.tsx

```diff
@@ -30,18 +26,10 @@ describe("ZoneSelect", () => {
   });
 
   it("disables select if zones have not loaded", () => {
-    const state = factory.rootState({
-      zone: factory.zoneState({
-        genericActions: factory.zoneGenericActions({ fetch: "idle" }),
-      }),
-    });
-    const store = mockStore(state);
-
-    renderWithMockStore(
+    renderWithProviders(
```

Comment:
IMO, you should use `mockIsPending()` before rendering the component, as that will force useQuery `isPending` to be true and simulate loading